go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/segmentio/stats
+module github.com/InVisionApp/stats
```

Comment:
@mnichols it looks like you need to revert this part

Reply:
ah nice catch...fixed! @maxcnunes